sound/soc/sof/sof-pci-dev.c

```diff
@@ -128,7 +128,7 @@ static const struct sof_dev_desc cfl_desc = {
 	.chip_info = &cnl_chip_info,
 	.default_fw_path = "intel/sof",
 	.default_tplg_path = "intel/sof-tplg",
-	.nocodec_fw_filename = "sof-cnl.ri",
+	.nocodec_fw_filename = "sof-cfl.ri",
```

Comment:
for cfl,   we use sof-cnl.ri for HDA platform, why we need to use sof-cfl.ri for nocodec ?
I think we don't need to change here.

Reply:
if we start using a different firmware for all platforms, why would we leave CFL and CNL the same? That'd be the exception, so let's clean this up.